shared/chat/conversation/input-area/normal/platform-input.desktop.js

```diff
@@ -3,6 +3,7 @@
 import * as React from 'react'
 import * as Kb from '../../../../common-adapters'
 import * as Styles from '../../../../styles'
+import animationData from '../../../../common-adapters/animation-data.json'
```

Comment:
this should go back. we're going to have more animation stuff in the future

Reply:
ok